dir.props

```diff
@@ -69,13 +69,15 @@
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == ''">false</Optimize>
     <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
-    <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
+    <DefineConstants Condition="'$(MSBuildProjectExtension)' != '.vbproj'">$(DefineConstants);DEBUG;TRACE</DefineConstants>
```

Comment:
It looks like C# compiler handles comma separated defines, I'll switch to using commas for both languages instead.